snippets/product-grid-item.liquid

```diff
@@ -45,11 +45,15 @@
             </button>
              {% endif %}
             {% if settings.product_compare_enabled %}
-                <button class="btn btn-secondary" title="{{ 'products.compare_list.add_to_compare' | t }}" style="width: {% if product.available %}25{% else %}100{% endif %}%;" ng-click="addProductToCompareList('{{ product.id }}', $event)" ng-controller="productCompareListController">
-                    <span class="fa fa-bar-chart"></span>
-                </button>
-            {% endif %}
-        </div>
+                {% if product.available %}
```

Comment:
Need to get of rid the inline styles this should be made with css way only instead.